src/commands/new.ts

```diff
@@ -154,13 +155,14 @@ export default {
       .replace(/hello-world/g, projectNameKebab)
     let packageJson = JSON.parse(packageJsonRaw)
 
+    const merge = require("deepmerge-json")
```

Comment:
I'd probably move this back into the `if` statement below, since it's not needed anywhere except there.

Reply:
We can do this later. I'm merging.